### ISSUE TITLE
get rid of -auto stuff ?

### ISSUE THREAD

**Issue by hpeyerl** [nekromant/esp8266-frankenstein#85]
Just a thought... Maybe get rid of sta-auto* and telnet-autostart because it can be done in $bootcmd?

**Comment by nekromant** [comment]
Agreed, will clean up as soon as I get my hands on it, and we still need a working bootdelay.

**Comment by hpeyerl** [comment]
I can submit some changes that you can merge at your convenience... I haven't been following along, what's up with bootdelay?

**Comment by nekromant** [comment]
Same thing uboot does. In short - timeout before bootcmd is executed. If a user hits a button - we drop to prompt without executing bootcmd. Use case: Imagine there's something in bootcmd that screws up things and causes a fault.

**Comment by hpeyerl** [comment]
oh I see.. As in "we don't have one"... :)
